includes/class-wcpdf-settings.php

```diff
@@ -746,6 +747,21 @@ public function move_setting_after_id( $settings, $insert_settings, $after_setti
 		return $new_settings;
 	}
 
+	/**
+	 * Checks if guest access is enabled
+	 * 
+	 * @return bool
+	 */
+	public function is_guest_access_enabled() {
+		// return bool with filter applied
+		$guest_access = isset( $this->debug_settings['guest_access'] ) ? true : false;
+
+		$value_guest_access = apply_filters( 'wpo_wcpdf_guest_access_enabled', $guest_access );
+
+		return $guest_access;
```

Comment:
I think you can move the filter to this place instead of wrapping it `$value_guest_access`

Reply:
Done